lib/appium_lib/device/device.rb

```diff
@@ -18,7 +18,10 @@ module Device
         device_time:            'session/:session_id/appium/device/system_time',
         current_activity:       'session/:session_id/appium/device/current_activity',
         current_context:        'session/:session_id/context',
-        get_network_connection: 'session/:session_id/network_connection'
+        get_network_connection: 'session/:session_id/network_connection',
```

Comment:
if these endpoints are available in the official selenium client, why not just update the version of that which we depend on in this library, so we get them for free? maybe i'm not understanding the point of this change